dist/index.html

```diff
@@ -265,7 +266,30 @@
 			</select>
 
 			<input type="submit" name="" id="submit" value="Get Weather">
-		</form>
+		</form> <br>
+
+		<div class="weather-data">
+			<div class="row headers">
+				<div class="col-md-4 town-name  ">
+
+				</div>
+				<div class="col-md-4 degrees mr-4">
+					
+				</div>
+				<div class="col-md-3 sunset ">
+					
+				</div>
+			</div>
+			<div class="row datas">
+				<div class="col-md-3 sys"></div>
+				<div class="col-md-4 main-data"></div>
+				<div class="col-md-4 weather"></div>
+				
+			</div>
```

Comment:
let's hide this section till there is weather data to be displayed.

Reply:
done